be/src/http/download_action.cpp

```diff
@@ -123,7 +123,7 @@ void DownloadAction::handle(HttpRequest* req) {
         handle_normal(req, file_path);
     }
 
-    LOG(INFO) << "deal with download requesst finished! ";
+    LOG(INFO) << "deal with download request finished! ";
```

Comment:
this log message can be improved, should add more information or just remove it?